matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/extensions/MetricsExtensions.kt

```diff
@@ -17,12 +17,16 @@
 package org.matrix.android.sdk.api.extensions
```

Comment:
I wonder why this class is called `MetricsExtensions` and is in the package `extensions`? It seems to me there is no extension methods in this file. Maybe we should rename and move it somewhere else?

Reply:
Yeah, you are right. Initially, I was thinking of making them as extension of `MetricsPlugin` but now they are just functions so we need to rename it.